source/operations/install-deploy-manage/deploy-minio-tenant.rst

```diff
@@ -110,442 +110,528 @@ volume and a supporting
+           Run ``kubectl create ns <new_namespace>`` to add one.
+
+         For example, the following command creates a new tenant with 16 volumes, total capacity of 16 Tebibytes, using the latest MinIO image, on four servers, in the ``minio`` namespace.
+
+         .. code-block:: shell
+            :class: copyable
+
+            kubectl minio tenant create minio_tenant --capacity 16Ti --servers 4 --volumes 16 --namespace minio
+
+      2) Add any optional configurations
+
+         You can further customize your tenant while executing the command by including any or all of the following *optional* flags.
+
+         - :mc:`~kubectl minio tenant create --image`
+           
+           Customize the ``minio`` image to use.
+  
+           By default, the Operator uses the latest ``minio`` release image when creating the tenant.
+           To specify a different version, use the :mc:`~kubectl minio tenant create --image` flag.
+
+           See the `MinIO Quay <https://quay.io/repository/minio/minio>`__ or the `MinIO DockerHub <https://hub.docker.com/r/minio/minio/tags>`__ repositories for a list of valid tags.
+
+         - :mc:`~kubectl minio tenant create --image-pull-secret`
+           
+           If required, specify the secret to use when pulling the ``minio`` image.
+
+           Use :mc:`~kubectl minio tenant create --image-pull-secret` to specify the secret.
+
+         - :mc:`~kubectl minio tenant create --kes-config`
+           
+           Configure a :minio-git:`Key Encrption Service (KES) <kes>`
+
+           Use the :mc:`~kubectl minio tenant create --kes-config` flag to specify the name of the secret to use for KES Key Management Service (KMS) setup.
+
+           Enabling Server Side Encryption (SSE) also deploys a MinIO :minio-git:`KES <kes>` service in the Tenant to faciliate SSE operations.
+  
+           For more, see the `Github documentation <https://github.com/minio/kes/wiki>`__.
+
+         - :mc:`~kubectl minio tenant create --storage-class`
+           
+           Specify the storage class to use.
+
+           New MinIO tenants use the ``default`` storage class.
+           To specify a different storage class, add the :mc:`~kubectl minio tenant create --storage-class` flag.
+
+           The specified :mc-cmd:`~kubectl minio tenant create --storage-class` *must* match the ``storage-class`` of the Persistent Volumes (``PVs``) to which the ``PVCs`` should bind.
+
+           MinIO strongly recommends creating a Storage Class that corresponds to locally-attached volumes on the host machines on which the Tenant deploys. 
+           This ensures each pod can use locally-attached storage for maximum performance and throughput. 
+           See the :ref:`Deploy MinIO Tenant <minio-k8s-deploy-minio-tenant>` tutorial for guidance on creating Storage Classes for supporting the MinIO Tenant.
+
+      3) Generate a YAML file for further customizations
+
+         Other options exist for customizing a MinIO tenant.
```

Comment:
https://github.com/minio/operator/blob/master/docs/crd.adoc

So MinIO Operator installs a Custom Resource Definition for describing Tenants. Advanced users can generate a base YAML using the CLI and customize based on the CRD.

Reply:
I'll have to read up on this. For now, I'll just essentially borrow this language and insert it.